gammapy/catalog/hawc.py

```diff
@@ -1,15 +1,16 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 """HAWC catalogs (https://www.hawc-observatory.org)."""
+import abc
 import numpy as np
 from astropy.table import Table
 from gammapy.modeling.models import Model, SkyModel
 from gammapy.utils.scripts import make_path
 from .core import SourceCatalog, SourceCatalogObject
 
-__all__ = ["SourceCatalog2HWC", "SourceCatalogObject2HWC"]
+__all__ = ["SourceCatalog2HWC", "SourceCatalog3HWC"]
```

Comment:
Here you also need to add `SourceCatalogObject2HWC` and `SourceCatalogObject3HWC` to expose them on the global API.

Reply:
done